src/Components/Web.JS/src/Rendering/RendererEventDispatcher.ts

```diff
@@ -10,9 +10,9 @@ export function dispatchEvent(eventDescriptor: EventDescriptor, eventArgs: UIEve
     throw new Error('eventDispatcher not initialized. Call \'setEventDispatcher\' to configure it.');
   }
 
-  return eventDispatcherInstance(eventDescriptor, eventArgs);
+  eventDispatcherInstance(eventDescriptor, eventArgs);
 }
 
-export function setEventDispatcher(newDispatcher: (eventDescriptor: EventDescriptor, eventArgs: UIEventArgs) => Promise<void>): void {
+export function setEventDispatcher(newDispatcher: (eventDescriptor: EventDescriptor, eventArgs: UIEventArgs) => void): void {
```

Comment:
I've changed `newDispatcher` to return `void` because we only call it in one place, and that place discards the returned value anyway. Event-triggering is fire-and-forget from the JS side, because any errors are handled on the .NET side.

So having it return `void` is a more accurate representation of what's going on, and it simplifies a few other bits of the code.